notebooks/4_1_combinatorial_library_data.ipynb

```diff
@@ -59,11 +59,7 @@
     "from pathlib import Path\n",
```

Comment:
<p>I don't quite understand this number - it's just the number of ligands right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/KinFragLib/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>yes</p>